backend/api/communication/BUILD

```diff
@@ -11,6 +11,7 @@ app_deps = [
     "//backend:channel",
     "//backend:metadata",
     "//backend:read-receipt",
+    "//lib/java/jwt:airy-jwt",
```

Comment:
If you make `spring-auth` export `//lib/java/jwt:airy-jwt` (usually we avoid this, but here it seems ok) you only have to change the dependencies of the test library 🙏

Reply:
NICE 👍  TIL